Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,372 @@
+indent: true
+---
+{% include_relative branch.liquid %}
+
+## Ceph Dashboard Monitoring
+
+To enable monitoring and alerting functionality for the Ceph dashboard, additional steps are required to install the required dependencies.
+
+The monitoring stack components required include:
+
+1. [Prometheus operator](#prometheus-operator): records real-time metrics in a time series database
+2. [Prometheus Ceph manager module](#prometheus-ceph-manager-module): provide Ceph cluster metrics to Prometheus
+3. [Grafana](#grafana): displays charts for the information collected by Prometheus
+4. [Prometheus Node exporter](#prometheus-node-exporter): provides cluster nodes machine metrics to Prometheus
+5. [Prometheus alert manager](#prometheus-alert-manager): manage alerts produced in Prometheus
+
+## Production systems considerations
+
+The documentation provided is intended to explain how to provide the monitoring and alerting capabilities to the Ceph Dashboard. This not is replacing in any way the original procedures/documentation that are needed to install the whole 
+monitoring stack in a kubernetes production system. Security, High availability, Multi Tenancy aspects are not in the scope of this document and those topics must be managed following the recommendations/documentation of each monitoring stack component.
+
+Usually the monitoring stack installed in a Kubernetes production system is a complete separate functional component that provides service not only to Ceph, but also to other applications. if that is the case, then the enablement of  the monitoring and alerting functionality in the Ceph Dashboard is reduced only to two simple steps:
+
+1. Enable dashboard, prometheus manager module and monitoring in the [Ceph cluster CRD](#ceph-cluster).
+2. Use the [Rook toolbox](ceph-toolbox.md) to set the urls used by the different monitoring stack components, and to disable (_set-*-api-ssl-verify settings_) the ssl certificates verification if self signed certificates are used. 
+```console
+ceph dashboard set-grafana-api-url <grafana_url>
+ceph dashboard set-grafana-api-ssl-verify <False|True>
+ceph dashboard set-alertmanager-api-host <alert manager url>
+ceph dashboard set-alertmanager-api-ssl-verify <False|True>
+ceph dashboard set-prometheus-api-host <prometheus url>
+ceph dashboard set-prometheus-api-ssl-verify <False|True>
+```
+
+## Preparation
+
+Clone the Rook repository to have access to the different resource files used in the next steps:
+```console
+git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+
+## Ceph Cluster
+
+Dashboard, prometheus manager modules and Prometheus monitoring must be enabled in your cluster CRD:
+```yaml
+mgr:
+...
+  modules:
+    ...
+  - name: prometheus
+    enabled: true
+...
+dashboard:
+  enabled: true
+  ssl: true
+...
+monitoring:
+  enabled: true
+  rulesNamespace: rook-ceph  
+```
+
+## Prometheus Operator
```

Comment:
Does prometheus need to be installed before the ceph dashboard configured? Or the order is not important?

Reply:
Not important, really the ceph dashboard get the information from prometheus Through Grafana